appinfo/info.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<info>
+	<id>oc-web</id>
+	<name>ownCloud Web</name>
+	<summary>Next generation user interface for ownCloud</summary>
+	<description>
+    Next generation user interface for ownCloud based on vue.js.
+    This is an early release with known limitations and missing features.
+
+    Please, report any findings in https://github.com/owncloud/phoenix/issues
+  </description>
+	<licence>AGPLv3</licence>
+	<author>ownCloud</author>
+	<version>1.0.0</version>
```

Comment:
If we decide against releasing version `1.0.0` together with this I'll change this :wink: